test/cypress/integration/stripe-setup/responsible_person_spec.js

```diff
@@ -0,0 +1,518 @@
+'use strict'
+
+const commonStubs = require('../../utils/common_stubs')
+
+describe('Stripe setup: responsible person page', () => {
+  const gatewayAccountId = 42
+  const userExternalId = 'userExternalId'
+
+  const firstName = 'William'
+  const typedFirstName = 'William '
+  const lastName = 'Benn'
+  const typedLastName = ' Benn'
+  const addressLine1 = '52 Festive Road'
+  const typedAddressLine1 = ' 52 Festive Road'
+  const addressLine2 = 'Putney'
+  const typedAddressLine2 = 'Putney '
+  const city = 'London'
+  const typedCity = 'London '
+  const postcode = 'SW15 1LP'
+  const typedPostcode = 'sw151lp '
+  const dobDay = '25'
+  const typedDobDay = '25 '
+  const dobMonth = '02'
+  const typedDobMonth = ' 02'
+  const dobYear = '1971'
+  const typedDobYear = '1971 '
+  const friendlyDob = '25 February 1971'
+  const longText = 'This text is 300 ...............................................................................' +
+    '...............................................................................................................' +
+    '............................................................................ characters long'
+
+  const stubStripeAccountGet = function stubStripeAccountGet (stripeAccountId) {
```

Comment:
Better naming 👍 Probably we should use similar in the other tests as well

Reply:
It dawned on me that “stub” can be used as a verb and then I was off to the races!